Source/Scene/Model.js

```diff
@@ -772,6 +781,10 @@ define([
         return JSON.parse(json);
     }
 
+    Model.getDefaultCacheKey = function(url) {
```

Comment:
I'm calling it in `ModelInstanceCollection` as well. I'm not really happy with this either.